scripts/scil_visualize_seeds.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Visualize seeds where streamlines originate from
+in a tractogram
+"""
+
+import argparse
+
+from fury import window, actor
+from nibabel.streamlines import detect_format, TrkFile
+
+from scilpy.io.utils import (
+    add_overwrite_arg,
+    assert_inputs_exist,
+    assert_outputs_exists)
+
+
+def _build_args_parser():
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('tractogram', help='Tractogram file')
+    parser.add_argument('--dest', type=str, help='If set, save a ' +
```

Comment:
I don't like the option name. Maybe rename --save because for me dest is the path without the filename